plugins/okta/komand_okta/util/helpers.py

```diff
@@ -12,3 +14,26 @@ def clean(item_to_clean: Union[dict, list]) -> Union[dict, list]:
 
 def get_hostname(hostname: str) -> str:
     return hostname.replace("https://", "").replace("http://", "")
+
+
+def validate_url(url: str) -> bool:
+    """
+    Check domain against allowedHost list for Okta.
+    Return True if Okta URL is valid, False otherwise
+    :param url: The URL to check
+    :type: str
+
+    :return: boolean indicating if URL is valid
+    :rtype: bool
+    """
+    if not url.startswith(("http://", "https://")):
+        prefix = "https://"
+        url = prefix + url
```

Comment:
is this part redundant since in the connection you're adding `https://` to the URL before calling this method?

Reply:
was thinking that but I've left it in here just incase as it's a util function it may be used somewhere else 

without the prefix the urlparse has unexpected behaviour and doesn't identify the netloc correctly. can't hurt to leave it in but can remove it if it really doesn't make sense

Reply:
I would double check to see if this method is used anywhere else in the file, if its not then im wondering if this code ever gets hit at all

Reply:
It's not used anywhere else since I added it in this PR, i've left the prefix stuff incase it's used by someone else in future. It's a simple check which will ensure the function works as intended